build-system/tasks/compile-bind-expr.js

```diff
@@ -23,7 +23,12 @@ gulp.task('compile-bind-expr', function() {
   const path = 'extensions/amp-bind/0.1/';
 
   const bnf = fs.readFileSync(path + 'bind-expr-impl.jison', 'utf8');
-  const settings = {type: 'lalr', debug: false, moduleType: 'js'};
+  const settings = {
```

Comment:
could we share this option somewhere instead of repeating it?

Reply:
We'd need to add a helper file for all three `compile-***-expr.js` files. I'll file this as a separate issue.